### ISSUE TITLE
Update librispeech to use num_utt2frames, avoiding the use of wav-to-duration

### ISSUE THREAD

**Issue by kkm000** [kaldi-asr/kaldi#3610]
The dataset is both popular and large. This is worth doing. 

https://groups.google.com/forum/#!topic/kaldi-help/4gCnj9K5X2k
```
local/data_prep.sh
utils/data/get_utt2dur.sh: segments file does not exist so getting durations from wave files 
utils/data/get_utt2dur.sh: could not get utterance lengths from sphere-file headers, using wav-to-duration
```
This may slash the prep time in half in an extreme case, if the source disk with the .flac files is slow.

At the same time, get rid of the dependency on flac, as sox handles flac files just fine.

**Comment by danpovey** [comment]
I'd be open to hosting that file (utt2dur) on OpenSLR in Librispeech's own directory as an extra file, if that would make things easier.  (i.e. if it's not available from any header info.)

**Comment by danpovey** [comment]
Fixed in #3620
